app/views/contributions/show.html.erb

```diff
@@ -7,8 +7,13 @@
     <p class="subtitle"><%= contribution.description %></p>
     <% if contribution.matched? %>
       <i class="far fa-check-circle has-text-primary is-size-5 has-text-weight-semibold"><span class="ml-1">Claimed</span></i>
-    <% else %>
-      <%= button_to "Claim This #{contribution.type}", new_contribution_claim_path(contribution), class: "button is-primary", method: :get %>
+    <% else  %>
+      <% if contribution.has_email? %>
+        <%= button_to "Claim This #{contribution.type}", new_contribution_claim_path(contribution), class: "button is-primary", method: :get %>
+      <% else %>
+        <%= button_to "Claim This #{contribution.type}", new_contribution_claim_path(contribution), class: "button is-primary", disabled: true, method: :get %>
+        <p class="is-6 is-italic">An ask without an email address cannot be claimed in peer-to-peer mode</p>
+      <% end %>
```

Comment:
What do y'all think of this change to add explanatory text next to the button when disabled? Is there a cleaner way to do this without another conditional? 

Also, I added `is-6` and `is-italic` classes because I saw those on the bulma docs. I looked through our styling and didn't see anything related to explanatory-type formatting.

Reply:
Looks good @leahriffell . I don't recall anything in bulma specifically for this kind of thing; i think a `p` with those classes is very reasonable.